### PR TITLE
Project 1

### PR DESCRIPTION
**Pull request by IamYunusAli**
In these pull requests me and my partner Godfrey-Alimony have implemented.

use plain javascript to implement the booklist 

-  Add the book to the list
- remove from the list
- and Show a list of Books on the front page of the list  

you can check the demo [here](https://iamyunusali.github.io/Awesome-books/)

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+function listBooks() {
+  const books = JSON.parse(window.localStorage.getItem('BookList'));
+  books.forEach((book) => {
+    bookContainer.innerHTML += `
+        <article>
+        <p class="book-title">${book.title}</p>
+        <p class="book-author">${book.author}</p>
+        <button class="removeBtn">Remove</button>
+        <hr>
+        </article>`;
```
Comment (sja-thedude) [inline script.js]: - [x] Good job so far in adding and removing books correctly. However, there are undefined values which are persistent even after deleting it. It is not needed kindly remove it.

https://user-images.githubusercontent.com/70269507/175964237-1dd176e3-cb1b-476c-b775-d9c9c2296b68.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @IamYunusAli & @Godfrey-Alimony, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional readme
✔️ All linters passing
✔️ Good PR Description

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (IamYunusAli) [reply script.js]: you can check the hosted one here [link](https://iamyunusali.github.io/Awesome-books/)

File: script.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  const newBook = { title, author };
+  books.push(newBook);
+  window.localStorage.setItem('BookList', JSON.stringify(books));
+}
+
+function removeBook(title) {
+  const books = JSON.parse(window.localStorage.getItem('BookList'));
+  const keptBooks = books.filter((book) => book.title !== title);
+  window.localStorage.setItem('BookList', JSON.stringify(keptBooks));
+}
```
Comment (danmainah) [inline script.js:18]: @Team, great job implementing the changes requested.

**Review by danmainah — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**